src/test/java/ua/softserveinc/tc/validator/EmailValidatorTest.java

```diff
@@ -0,0 +1,119 @@
+package ua.softserveinc.tc.validator;
+
+/**
+ * Created by Valery on 16.11.2016.
```

Comment:
add javadoc for this Test.
This should contain information about what are you testing here.